tests/sentry/search/django/tests.py

```diff
@@ -60,41 +61,29 @@ def setUp(self):
             message='bar',
             times_seen=10,
             status=GroupStatus.RESOLVED,
-            last_seen=datetime(2013, 7, 14, 3, 8, 24, 880386),
-            first_seen=datetime(2013, 7, 14, 3, 8, 24, 880386),
+            last_seen=datetime(2013, 7, 14, 3, 8, 24, 880386, tzinfo=pytz.utc),
+            first_seen=datetime(2013, 7, 14, 3, 8, 24, 880386, tzinfo=pytz.utc),
             score=ScoreClause.calculate(
                 times_seen=10,
-                last_seen=datetime(2013, 7, 14, 3, 8, 24, 880386),
+                last_seen=datetime(2013, 7, 14, 3, 8, 24, 880386, tzinfo=pytz.utc),
             ),
         )
 
         self.event2 = self.create_event(
             event_id='b' * 32,
             group=self.group2,
-            datetime=datetime(2013, 7, 14, 3, 8, 24, 880386),
+            datetime=datetime(2013, 7, 14, 3, 8, 24, 880386, tzinfo=pytz.utc),
             tags={
                 'server': 'example.com',
                 'environment': 'staging',
                 'url': 'http://example.com',
             }
         )
 
-        for key, value in self.event1.data['tags']:
```

Comment:
would this test no longer pass if Environments were not setup?

(i ask because reviewing I'd prefer to see 'existing functionality still works when it needs to')

Reply:
Yeah, when using the legacy/v1 `tagstore` backend the functionality should be equivalent (since that backend doesn't do any environment filtering anyway.)